### DISCUSSION TITLE
Running BirdNET pi on battery

### DISCUSSION THREAD

**Discussion by PeterIP4** [mcguirepr89/BirdNET-Pi#493] [Q&A] [answered]
Currently I am running BirdNET pi using a battery which lasts about 20 hours.

Can anyone suggest ways to extend battery life such as turning off Bluetooth?

Thanks in anticipation.

**Comment by mcguirepr89** [comment, marked as answer]
This may also prove helpful: https://github.com/mvp/uhubctl

**Comment by NIDcornifleur** [comment]
https://learn.pi-supply.com/make/how-to-save-power-on-your-raspberry-pi/

**Reply by PeterIP4** [reply]
I’ll try those out. I bought a meter so I can see the result of the changes. Thanks
